oop/homework.py

```diff
@@ -23,179 +24,270 @@ class Cat:
 
     * Implement method run it receives hours value
       Calculate run km per hours remember that you have average_speed value
-      Than if your cat run less or eq than 25 _reduce_saturation_level with value 2
-      if it runs between 25(not including) and 50(including) than _reduce_saturation_level with value 5
-      if it runs between 50(not including) and 100(including) than _reduce_saturation_level with value 15
-      if it runs between 100(not including) and 200(including) than _reduce_saturation_level with value 25
-      if it runs more than 200(not including) than _reduce_saturation_level with value 50
-
+      Than if your cat run less or eq than 25
+      _reduce_saturation_level with value 2
+      if it runs between 25(not including) and 50(including)
+      than _reduce_saturation_level with value 5
+      if it runs between 50(not including) and 100(including)
+      than _reduce_saturation_level with value 15
+      if it runs between 100(not including) and 200(including)
+      than _reduce_saturation_level with value 25
+      if it runs more than 200(not including)
+      than _reduce_saturation_level with value 50
       return text like this: f"Your cat ran {ran_km} kilometers"
 
     * Implement get_saturation_level and return saturation_level
-      if saturation_level eq 0 return text like this: "Your cat is dead :("
+      if saturation_level eq 0 return text like this: "Your cat is died :("
 
     * Implement get_average_speed and return average_speed
-
     """
 
+    product_dict = {
+        'fodder': 10,
+        'apple': 5,
+        'milk': 2
+    }
+
     def __init__(self, age):
-        pass
+        self.age = age
+        self.saturation_level = 50
+        self.average_speed = self._set_average_speed()
 
     def eat(self, product):
-        pass
+        self._increase_saturation_level(self.product_dict.get(product, 0))
```

Comment:
для кращої читабельності бажано додати якусь проміжну змінну, наприклад
```suggestion
        few_points = self.product_dict.get(product, 0)
        self._increase_saturation_level(few_points)
```